<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>716</width>
    <height>534</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>690</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Harmonic</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QCustomPlot" name="widget" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>350</width>
            <height>350</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_2" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>317</width>
            <height>321</height>
           </size>
          </property>
          <widget class="QRadioButton" name="radioButton">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>10</y>
             <width>181</width>
             <height>17</height>
            </rect>
           </property>
           <property name="mouseTracking">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>случайная генерация данных </string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
           <property name="autoRepeat">
            <bool>false</bool>
           </property>
          </widget>
          <widget class="QRadioButton" name="radioButton_2">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>30</y>
             <width>71</width>
             <height>17</height>
            </rect>
           </property>
           <property name="mouseTracking">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Заданые</string>
           </property>
          </widget>
          <widget class="QDoubleSpinBox" name="doubleSpinBox_6">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>60</y>
             <width>49</width>
             <height>20</height>
            </rect>
           </property>
           <property name="contextMenuPolicy">
            <enum>Qt::DefaultContextMenu</enum>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="inputMethodHints">
            <set>Qt::ImhFormattedNumbersOnly</set>
           </property>
           <property name="wrapping">
            <bool>false</bool>
           </property>
           <property name="frame">
            <bool>true</bool>
           </property>
           <property name="specialValueText">
            <string/>
           </property>
           <property name="correctionMode">
            <enum>QAbstractSpinBox::CorrectToPreviousValue</enum>
           </property>
           <property name="stepType">
            <enum>QAbstractSpinBox::DefaultStepType</enum>
           </property>
          </widget>
          <widget class="QDoubleSpinBox" name="doubleSpinBox_5">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>90</y>
             <width>49</width>
             <height>20</height>
            </rect>
           </property>
          </widget>
          <widget class="QDoubleSpinBox" name="doubleSpinBox_8">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>120</y>
             <width>49</width>
             <height>20</height>
            </rect>
           </property>
          </widget>
          <widget class="QDoubleSpinBox" name="doubleSpinBox">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>150</y>
             <width>49</width>
             <height>20</height>
            </rect>
           </property>
          </widget>
          <widget class="QDoubleSpinBox" name="doubleSpinBox_7">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>180</y>
             <width>49</width>
             <height>20</height>
            </rect>
           </property>
          </widget>
          <widget class="QDoubleSpinBox" name="doubleSpinBox_2">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>210</y>
             <width>49</width>
             <height>20</height>
            </rect>
           </property>
          </widget>
          <widget class="QRadioButton" name="radioButton_4">
           <property name="geometry">
            <rect>
             <x>30</x>
             <y>250</y>
             <width>36</width>
             <height>17</height>
            </rect>
           </property>
           <property name="text">
            <string>sin</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup</string>
           </attribute>
          </widget>
          <widget class="QRadioButton" name="radioButton_3">
           <property name="geometry">
            <rect>
             <x>30</x>
             <y>280</y>
             <width>39</width>
             <height>17</height>
            </rect>
           </property>
           <property name="text">
            <string>cos</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup</string>
           </attribute>
          </widget>
          <widget class="QLabel" name="label_2">
           <property name="geometry">
            <rect>
             <x>80</x>
             <y>60</y>
             <width>31</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>min</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_3">
           <property name="geometry">
            <rect>
             <x>80</x>
             <y>90</y>
             <width>31</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>max</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_7">
           <property name="geometry">
            <rect>
             <x>80</x>
             <y>120</y>
             <width>31</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>min</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_8">
           <property name="geometry">
            <rect>
             <x>80</x>
             <y>150</y>
             <width>31</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>max</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_4">
           <property name="geometry">
            <rect>
             <x>80</x>
             <y>180</y>
             <width>31</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>min</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_5">
           <property name="geometry">
            <rect>
             <x>80</x>
             <y>210</y>
             <width>31</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>max</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_9">
           <property name="geometry">
            <rect>
             <x>150</x>
             <y>130</y>
             <width>121</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Начальная фаза (рад)</string>
           </property>
          </widget>
          <widget class="QLabel" name="label">
           <property name="geometry">
            <rect>
             <x>150</x>
             <y>70</y>
             <width>81</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Амплитуда (м)</string>
           </property>
          </widget>
          <widget class="QCheckBox" name="checkBox">
           <property name="geometry">
            <rect>
             <x>210</x>
             <y>10</y>
             <width>91</width>
             <height>17</height>
            </rect>
           </property>
           <property name="text">
            <string>ограничить</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_6">
           <property name="geometry">
            <rect>
             <x>140</x>
             <y>190</y>
             <width>161</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Циклическая частота (рад/с)</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_10">
           <property name="geometry">
            <rect>
             <x>100</x>
             <y>270</y>
             <width>61</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Функция</string>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Car</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QCustomPlot" name="widget_3" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>350</width>
            <height>350</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_4" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>320</width>
            <height>365</height>
           </size>
          </property>
          <widget class="QDoubleSpinBox" name="doubleSpinBox_3">
           <property name="geometry">
            <rect>
             <x>30</x>
             <y>160</y>
             <width>62</width>
             <height>22</height>
            </rect>
           </property>
          </widget>
          <widget class="QDoubleSpinBox" name="doubleSpinBox_9">
           <property name="geometry">
            <rect>
             <x>30</x>
             <y>190</y>
             <width>62</width>
             <height>22</height>
            </rect>
           </property>
          </widget>
          <widget class="QDoubleSpinBox" name="doubleSpinBox_10">
           <property name="geometry">
            <rect>
             <x>30</x>
             <y>220</y>
             <width>62</width>
             <height>22</height>
            </rect>
           </property>
          </widget>
          <widget class="QDoubleSpinBox" name="doubleSpinBox_11">
           <property name="geometry">
            <rect>
             <x>30</x>
             <y>250</y>
             <width>62</width>
             <height>22</height>
            </rect>
           </property>
          </widget>
          <widget class="QDoubleSpinBox" name="doubleSpinBox_12">
           <property name="geometry">
            <rect>
             <x>30</x>
             <y>280</y>
             <width>62</width>
             <height>22</height>
            </rect>
           </property>
          </widget>
          <widget class="QDoubleSpinBox" name="doubleSpinBox_13">
           <property name="geometry">
            <rect>
             <x>30</x>
             <y>310</y>
             <width>62</width>
             <height>22</height>
            </rect>
           </property>
          </widget>
          <widget class="QDoubleSpinBox" name="doubleSpinBox_14">
           <property name="geometry">
            <rect>
             <x>30</x>
             <y>100</y>
             <width>62</width>
             <height>22</height>
            </rect>
           </property>
          </widget>
          <widget class="QDoubleSpinBox" name="doubleSpinBox_15">
           <property name="geometry">
            <rect>
             <x>30</x>
             <y>130</y>
             <width>62</width>
             <height>22</height>
            </rect>
           </property>
          </widget>
          <widget class="QRadioButton" name="radioButton_5">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>10</y>
             <width>181</width>
             <height>17</height>
            </rect>
           </property>
           <property name="text">
            <string>случайная генерация данных </string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup_2</string>
           </attribute>
          </widget>
          <widget class="QRadioButton" name="radioButton_6">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>40</y>
             <width>82</width>
             <height>17</height>
            </rect>
           </property>
           <property name="text">
            <string>Заданые</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup_2</string>
           </attribute>
          </widget>
          <widget class="QCheckBox" name="checkBox_2">
           <property name="geometry">
            <rect>
             <x>210</x>
             <y>10</y>
             <width>91</width>
             <height>17</height>
            </rect>
           </property>
           <property name="text">
            <string>ограничить</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_13">
           <property name="geometry">
            <rect>
             <x>100</x>
             <y>100</y>
             <width>31</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>min</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_14">
           <property name="geometry">
            <rect>
             <x>100</x>
             <y>130</y>
             <width>31</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>max</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_15">
           <property name="geometry">
            <rect>
             <x>100</x>
             <y>160</y>
             <width>31</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>min</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_16">
           <property name="geometry">
            <rect>
             <x>100</x>
             <y>190</y>
             <width>31</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>max</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_17">
           <property name="geometry">
            <rect>
             <x>100</x>
             <y>220</y>
             <width>31</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>min</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_18">
           <property name="geometry">
            <rect>
             <x>100</x>
             <y>250</y>
             <width>31</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>max</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_19">
           <property name="geometry">
            <rect>
             <x>100</x>
             <y>280</y>
             <width>31</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>min</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_20">
           <property name="geometry">
            <rect>
             <x>100</x>
             <y>310</y>
             <width>31</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>max</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_21">
           <property name="geometry">
            <rect>
             <x>160</x>
             <y>110</y>
             <width>111</width>
             <height>21</height>
            </rect>
           </property>
           <property name="text">
            <string>уровень топлива</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_22">
           <property name="geometry">
            <rect>
             <x>160</x>
             <y>180</y>
             <width>101</width>
             <height>21</height>
            </rect>
           </property>
           <property name="text">
            <string>скорость (км/ч)</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_23">
           <property name="geometry">
            <rect>
             <x>160</x>
             <y>240</y>
             <width>81</width>
             <height>21</height>
            </rect>
           </property>
           <property name="text">
            <string>обороты</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_24">
           <property name="geometry">
            <rect>
             <x>160</x>
             <y>290</y>
             <width>141</width>
             <height>21</height>
            </rect>
           </property>
           <property name="text">
            <string>температура двигателя</string>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Plane</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout"/>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QDoubleSpinBox" name="doubleSpinBox_4">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>23</width>
          <height>15</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="stepType">
         <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_11">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>Шаг (сек)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>32</width>
          <height>15</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_12">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>Время симуляции (сек)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>Построить график</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>716</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup_2"/>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
